connectors/v2/youtube.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+/* global Connector */
+
+var scrobbleMusicOnly = false;
+chrome.storage.local.get('Connectors', function(data) {
```

Comment:
@janbaykara you're right. The storage abstraction is not injected into page. I don't think it's necessary and we should probably inject as many scripts as possible (yet still convenient for connectors impl.)

I'm thinking about unified configuration getter for current connector somewhere in base connector, but this is totally ok now ;-)